src/Services/AllergyIntoleranceService.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+/**
+ * AllergyIntoleranceService
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Yash Bothra <yashrajbothra786gmail.com>
+ * @copyright Copyright (c) 2020 Yash Bothra <yashrajbothra786gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Services;
+
+use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Validators\AllergyIntoleranceValidator;
+use OpenEMR\Validators\ProcessingResult;
+
+require_once($GLOBALS['fileroot'] . '/custom/code_types.inc.php');
```

Comment:
general practice in classes is to use direct path so don't need to worry about if globals.php has been run
```
require_once(__DIR__  . '/../../custom/code_types.inc.php');
```